charts/localdev/values.yaml

```diff
@@ -182,6 +182,11 @@ portal:
       bpnDidResolver:
         # -- ApiKey for management endpoint of the bpnDidResolver. Secret-key 'bpndidresolver-api-key'.
         apiKey: ""
```

Comment:
Please also remove the apiKey as it will be removed with https://github.com/eclipse-tractusx/portal-backend/pull/1129

Reply:
bpnDidResolver -> apiKey has been removed